scipy/stats/mstats_basic.py

```diff
@@ -575,14 +595,26 @@ def _kendall_p_exact(n, c):
             if j <= c:
                 new[j:] -= new[:c+1-j]
         prob = np.sum(new)
+        p_mass_at_c = new[-1]/2
```

Comment:
Similar to above, but there was no explicit factor of `2` added above, so I figure we need to divide by `2` here.